unity/Assets/Maroon/scenes/experiments/MinimumSpanningTree/Scripts/MSTController.cs

```diff
@@ -0,0 +1,765 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System;
+using TMPro;
+using Maroon.UI; // for DialogueManager
+using GEAR.Localization; // for LanguageManager
+
+public class MSTConstants
+{
+    public const int MAX_Islands = 10;
+}
+
+// options for selecting start- and endislands manually
+enum ManualIslandPickerOptions
+{
+    noneSelected,
+    oneSelected,
+    bothSelected
+}
+
+public class MSTController : MonoBehaviour
+{
+    public GameObject IslandsParent;
```

Comment:
Please order the properties using the header attribute
https://docs.unity3d.com/ScriptReference/HeaderAttribute.html